src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -773,7 +781,9 @@ export const Details: React.FC<DetailsType> = ({
             ) : (
                 renderAppDetails()
             )}
-            {detailedStatus && <AppStatusDetailModal close={hideAppDetailsStatus} showAppStatusMessage={false} />}
+            {detailedStatus && (
+                <AppStatusDetailModal close={hideAppDetailsStatus} showAppStatusMessage={false} showConfigDriftInfo />
```

Comment:
showConfigDriftInfo should be based of fe-lib and to be verified for isolated airgapped